src/app/components/details/album_header.css

```diff
@@ -1,14 +1,21 @@
 .title-4 label {
-  font-size: 16px;
-  color: @theme_text_color;
+  color: @window_fg_color;
+  font-weight: bold;
+  text-decoration: none;
 }
 
 .title-4 {
   padding: 0;
 }
 
+
+.title-4:hover {
+  border-radius: 6px;
+  background-image: image(alpha(currentColor, 0.03));
```

Comment:
a bit too subtle maybe? I had not even noticed it before starting reading the code :D